cvm-python-app-remoteattest/attestationclientdeploy.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: attestation
+  labels:
+    app: attestation
+spec:
+  containers:
+  - image: lizzhaacr.azurecr.io/attestation-client:latest
+    imagePullPolicy: IfNotPresent
+    name: attestation-client
+    volumeMounts:
+      - mountPath: /sys/kernel/security
+        name: tcg
+      - mountPath: /dev/tpmrm0
+        name: tpmrm0
+    securityContext:
+      privileged: true
+  volumes:
+    - name: tcg
+      hostPath:
+        path: /sys/kernel/security
+    - name: tpmrm0
+      hostPath:
+        path: /dev/tpmrm0
+  restartPolicy: Always
```

Comment:
better to set restartPolicy to be "Never", if setting to "Always" the pod will end in CrashLoopBackoff state.

The latest yaml file:
https://github.com/lizzha/confidential-computing-cvm-guest-attestation/blob/9c58e46fc5600cc4cd1d84080933dde01b57a851/aks-linux-sample/cvm-attestation.yaml